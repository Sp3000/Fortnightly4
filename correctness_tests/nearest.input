nearest 0 0
place 0 0 red
nearest 0 0
nearest 1 1
nearest -1000 1000
place 1 0 green
nearest 0 0
nearest 1 0
nearest 0 1
nearest 1 1
place 0 4 purple
nearest 0 1
nearest 0 3
nearest 0 2
nearest 0 2
nearest 2 2
nearest -100 -100
nearest -50000 3
remove 0 0
nearest 0 2
nearest 0 0
nearest 1 2
remove 1 0
nearest 10000 10000
remove 0 4
nearest -1 -1
halt