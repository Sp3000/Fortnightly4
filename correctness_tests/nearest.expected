none
0 0 red
0 0 red
0 0 red
0 0 red
1 0 green
0 0 red
1 0 green
0 0 red
0 4 purple
0 0 red|0 4 purple
0 4 purple|0 0 red
1 0 green
0 0 red
0 4 purple
0 4 purple
1 0 green
1 0 green
0 4 purple
none